README.md

```diff
@@ -1 +1,49 @@
-# Hello-microverse-project-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> A simple project to demonstrate the GitHub flow at Microverse.
+
+**Hello Microverse** is a web page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> HAFED EFEHIJ
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/hafedEfheij)
+)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is MIT(./MIT.md) licensed.
```

Comment:
This is supposed to be a link, directing us to the `MIT.md` license file.

To fix the link, update this code line with this:
```
This project is [MIT](./MIT.md) licensed.
```

But simply adding this file would not fix the link. Because it would take you to the MIT.md file, which does not exist in your directory. So create a MIT.md file in the root directory and add this code there.
```
## Copyright 2021, YOUR_NAME

Permission is hereby granted, free of charge, to any person obtaining a copy of this _*PROJECT_NAME*_ and associated documentation files, to deal in the _*PROJECT_NAME*_ without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the _*PROJECT_NAME*_, and to permit persons to whom the _*PROJECT_NAME*_ is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the _*PROJECT_NAME*_.

THE _*PROJECT_NAME*_ IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE _*PROJECT_NAME*_ OR THE USE OR OTHER DEALINGS IN THE _*PROJECT_NAME*_.
```